src/context/RariContext.tsx

```diff
@@ -258,7 +258,7 @@ export const RariProvider = ({ children }: { children: ReactNode }) => {
   const isMobile = useIsMobile();
   useEffect(() => {
     if (!isMobile) {
-      login();
+      // login();
```

Comment:
I'd prefer to readd this. The flow of having to reconnect is unnecessary for must users who don't switch wallets that much (and if they do they are competent enough to disconnect imo)

Reply:
For any future readers we can't just uncomment this because then the auth modal would show up for first time user (which this PR is supposed to avoid). Current gameplan is to only call `login()` if `WEB3_CONNECT_CACHED_PROVIDER` is not undefined in localstorage (that way only returning users get logged in automatically)